src/main/kotlin/no/nav/syfo/pdl/api/PdlPersonApi.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.syfo.pdl.api
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.get
+import java.util.UUID
+import no.nav.syfo.dinesykmeldte.service.DineSykmeldteService
+import no.nav.syfo.pdl.service.PdlPersonService
+import no.nav.syfo.plugins.BrukerPrincipal
+import no.nav.syfo.util.getBrukerPrincipal
+import no.nav.syfo.util.getParam
+import no.nav.syfo.util.logger
+import no.nav.syfo.util.securelog
+
+private val log = logger("no.nav.syfo.pdl.api")
+
+fun Route.registerPdlPersonApi(
```

Comment:
Kan vurdere å kalle det noe med pilot evnt, siden det er API for å hente pilot-status?